fix: Remove userClassPathFirst properties (#355)
stackabletech/spark-k8s-operator: CHANGELOG.md

- Various documentation of the CRD ([#319]).
- [BREAKING] Removed version field. Several attributes have been changed to mandatory. While this change is
  technically breaking, existing Spark jobs would not have worked before as these attributes were necessary ([#319]).
- [BREAKING] Remove `userClassPathFirst` properties from `spark-submit`. This is an experimental feature that was
  introduced to support logging in XML format. The side effect of this removal is that the vector agent cannot
  aggregate output from the `spark-submit` containers. On the other side, it enables dynamic provisionining of

>   delta.io packages (among others) with Stackable stock images which is much more important. ([#355])

Reviewer: ```suggestion
  java packages (such as Delta Lake) with Stackable stock images which is much more important. ([#355])
```
